### PR TITLE
Gibbers now have a small chance to be a MEAT GRINDER

### PR DESCRIPTION
**Pull request by SgtHunk**
## About The Pull Request

Title, gibbers have a 5% chance to be a meat grinder, which changes it's name and description. Also changes mentions of "Gibber" in its code to [src] for more consistency.

<details> 
<summary>Why It's Good For The Game:</summary>
<br>

https://user-images.githubusercontent.com/68669754/202934270-bbd907b6-a1b4-4942-a897-a781e0894369.mp4

</details>

## Changelog

:cl:
add: A gibber, upon creation, might be a meat grinder. This does nothing meaningful, but at least one of your finger bones will still be intact.
/:cl:

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
why doesn't it make you a beefman

**Comment by SgtHunk** [comment]
> why doesn't it make you a beefman

is this maintainer approval to port beefmen

**Review by MrMelbert — APPROVED** [review]

**Comment by KittyNoodle** [comment]
make it so it only creates meat and eyeballs

File: code/modules/food_and_drinks/machinery/gibber.dm [hunk 1]
```diff
@@ -15,6 +15,12 @@
 
 /obj/machinery/gibber/Initialize(mapload)
 	. = ..()
+	if(prob(5))
+		name = "meat grinder"
+		desc = "Okay, if I... if I chop you up in a meat grinder, and the only thing that comes out, that's left of you, is your eyeball, \
+				you'r- you're PROBABLY DEAD! You're probably going to - not you, I'm just sayin', like, if you- if somebody were to, like, \
+				push you into a meat grinder, and, like, your- one of your finger bones is still intact, they're not gonna pick it up and go, \
+				Well see, yeah it wasn't deadly, it wasn't an instant kill move! You still got, like, this part of your finger left!"
```
Comment (Mothblocks) [inline code/modules/food_and_drinks/machinery/gibber.dm:23]: Only indent once

**Comment by calliophis-maculiceps** [comment]
meatgrinder.mp4

**Comment by Serxule** [comment]
i was going to post the jerma meme myself but then i realized its in the pr

File: code/modules/food_and_drinks/machinery/gibber.dm [hunk 2]
```diff
@@ -18,9 +18,9 @@
 	if(prob(5))
 		name = "meat grinder"
 		desc = "Okay, if I... if I chop you up in a meat grinder, and the only thing that comes out, that's left of you, is your eyeball, \
-				you'r- you're PROBABLY DEAD! You're probably going to - not you, I'm just sayin', like, if you- if somebody were to, like, \
-				push you into a meat grinder, and, like, your- one of your finger bones is still intact, they're not gonna pick it up and go, \
-				Well see, yeah it wasn't deadly, it wasn't an instant kill move! You still got, like, this part of your finger left!"
+		you'r- you're PROBABLY DEAD! You're probably going to - not you, I'm just sayin', like, if you- if somebody were to, like, \
```
Comment (Mothblocks) [inline code/modules/food_and_drinks/machinery/gibber.dm:21]: Indent ONCE, your last one indented twice

Comment (SgtHunk) [reply code/modules/food_and_drinks/machinery/gibber.dm:21]: goddamnit im a moron

File: code/modules/food_and_drinks/machinery/gibber.dm [hunk 3]
```diff
@@ -15,6 +15,12 @@
 
 /obj/machinery/gibber/Initialize(mapload)
 	. = ..()
+	if(prob(5))
+		name = "meat grinder"
+		desc = "Okay, if I... if I chop you up in a meat grinder, and the only thing that comes out, that's left of you, is your eyeball, \
+	you'r- you're PROBABLY DEAD! You're probably going to - not you, I'm just sayin', like, if you- if somebody were to, like, \
```
Comment (Mothblocks) [inline code/modules/food_and_drinks/machinery/gibber.dm:21]: im gonna cry

Comment (Mothblocks) [reply code/modules/food_and_drinks/machinery/gibber.dm:21]: indent ONCE

starting from desc

indent ONCE

Comment (SgtHunk) [reply code/modules/food_and_drinks/machinery/gibber.dm:21]: dont worry im crying too
doesnt this leave it literally how it was in the initial commit

Comment (SgtHunk) [reply code/modules/food_and_drinks/machinery/gibber.dm:21]: oh wow nvm initial commit was two indents from desc ok im dumb

**Comment by SgtHunk** [comment]
https://user-images.githubusercontent.com/68669754/203195671-27b14f1f-98b7-4d5d-a670-d7223de4f9f1.mp4

**Pull request merged**